website/docs/r/cloudant.html.markdown

```diff
@@ -0,0 +1,131 @@
+* `cors_config` - (Optional, Block List) Configuration for CORS. (see [below for nested arguments](#nestedblock--cors_config))
+  * Constraints: The minimum length is `1` item. Can conflict with `enable_cors` if it is set to `false`. In this case the CORS customization is not allowed.
+  * `allow_credentials` - (Optional, Boolean) Boolean value to allow authentication credentials. If set to true, browser requests must be done by using withCredentials = true.
+    * Constraints: The default value is `true`.
+  * `origins` - (Required, List of String) An array of strings that contain allowed origin domains. You have to specify the full URL including the protocol. It is recommended that only the HTTPS protocol is used. Subdomains count as separate domains, so you have to specify all subdomains used.
+* `enable_cors` - (Optional, Boolean) Boolean value to turn CORS on and off.
+  * Constraints: The default value is `true`. If it is set to `false`, then customizing `cors_config` is not allowed.
+* `environment_crn` - (Optional, Forces new resource, String) CRN of the IBM Cloudant Dedicated Hardware plan instance.
+* `id` - (Optional, String) The unique identifier of the new Cloudant resource.
+* `include_data_events` - (Optional, Boolean) Include `data` event types in events sent to IBM Cloud Activity Tracker with LogDNA for the IBM Cloudant instance. By default emitted events are only of `management` type.
+  * Constraints: The default value is `false`.
+* `legacy_credentials` - (Optional, Forces new resource, Boolean) Use both legacy credentials and IAM for authentication.
+  * Constraints: The default value is `false`.
+* `location` - (Required, Forces new resource, String) Target location or environment to create the resource instance.
+* `name` - (Required, String) A name for the resource instance.
+* `parameters` - (Optional, Forces new resource, Map) Arbitrary parameters to pass. Must be a JSON object.
+* `plan` - (Required, String) The plan type of the service.
+* `resource_group_id` - (Optional, Forces new resource, String) The resource group id.
+* `service_endpoints` - (Optional, String) Types of the service endpoints. Possible values are 'public', 'private', 'public-and-private'.
+* `tags` - (Optional, Set of String) Tags associated with the instance.
+
+## Attribute Reference
+
+In addition to all arguments above, you can access the following attribute references after your resource is created.:
+
+* `account_id` - (String) An alpha-numeric value identifying the account ID.
+* `allow_cleanup` - (Boolean) A boolean that dictates if the resource instance should be deleted (cleaned up) during the processing of a region instance delete call.
+* `created_at` - (String) The date when the instance was created.
+* `created_by` - (String) The subject who created the instance.
+* `crn` - (String) CRN of resource instance.
+* `dashboard_url` - (String) Dashboard URL to access resource.
```

Comment:
Dashboard URL to access resource. could be as

The dashboard URL to access resources.